### PR TITLE
[TESTENG-1557] Adding FeatureRequestSLA.

### PR DESCRIPTION
**Pull request by paulletang**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework/blob/master/CHANGELOG.md)

### CODE REVIEW

**Review by briantoms — DISMISSED** [review]

File: docs/user/FeatureRequestSLA.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Feature Request SLA
+
+This document describes the feature request process.
+
+## Feature Request Process
+
+1. A feature is requested using the issue tracker in either the [Relativity.Testing.Framework](https://github.com/relativitydev/relativity.testing.framework/issues/new?&labels=enhancement&template=feature_enhancement.yml) or [Relativity.Testing.Framework.Api](https://github.com/relativitydev/relativity.testing.framework.api/issues/new?&labels=enhancement&template=feature_enhancement.yml) repositories.
+2. A maintainer will follow up on the issue asking for more information if necessary.
+3. Feature Requests will be prioritized based on many criteria including, but not limited to: :+1: reactions, size, and estimated effort.
+4. The maintainers will work on feature requests as time permits. We encourage consumers of these repositories to [contribute](https://github.com/relativitydev/relativity.testing.framework/blob/master/CONTRIBUTING.md#code-contributions) where appropriate to foster a collaborative community.
```
Comment (paulletang) [inline docs/user/FeatureRequestSLA.md:10]: It's kinda vague, but given the upcoming team changes, this is about as much certainty as I feel comfortable committing to at this time. :/

Comment (ParimalPa) [reply docs/user/FeatureRequestSLA.md:10]: The maintainers will work on feature requests as _time & priority_ permits

Comment (paulletang) [reply docs/user/FeatureRequestSLA.md:10]: What change are you looking for with `SLA --> SLO`?

We could put together a SLO for the other page, something like 95% of P1's completed within the declared timeframe.
We don't really have anything measurable here though.

Comment (paulletang) [reply docs/user/FeatureRequestSLA.md:10]: - [ ] Add `priority` as a qualifier for working on feature requests
- [ ] Add `within our scope of influence` as a qualifier for working on feature requests

**Review by briantoms — APPROVED** [review]

**Pull request merged**